### ISSUE TITLE
Cannot GET remote.html

### ISSUE THREAD

**Issue by Linsanity77** [Jopyth/MMM-Remote-Control#70]
Every time I try to access the website for the module, I get:
Cannot GET /remote.html
Can someone please help me? Thanks

**Comment by ndom91** [comment]
Have you installed the installed the module? Via "npm install"?

**Comment by Linsanity77** [comment]
Yeah, I believe so. So I removed and quick installed MMM-Remote-Control again and when I tried to get to the website, it still said Cannot GET remote.html and I don't know why.

**Comment by ndom91** [comment]
Well all I can say is double check that you installed the module, that you have listed it in your config.js file correctly and if you have any crypto ticker / price modules installed, try disabling those. Both ethereum price and bitcoin price modules broke Remote-Control for me for some reason.

**Comment by Jopyth** [comment]
If you used the quick install this is an error you usually get, if you did not add the module to the `config.js` file.

**Comment by Linsanity77** [comment]
Here is part of my config file:
		},
                {
                        module: "compliments",
                        position: "lower_third",
                },
		{
			module: "MMM-Remote_Control",
			position: 'bottom_left',
		},
		{
			module: "currentweather",
			position: "top_right",

**Comment by Linsanity77** [comment]
In addition, I checked the console of my MagicMirror and I found this on it:
Load script: modules/MMM-Remote_Control//MMM-Remote_Control.js
Failed to load resource: the server responded with a status of 404 (Not Found)
Error on loading script: modules/MMM-Remote_Control//MMM-Remote_Control.js

I'm not sure what this means but maybe it's a clue as to what's wrong?

**Comment by Linsanity77** [comment]
Thanks I can't believe I didn't notice that. 😂

**Comment by SanSahInnovations** [comment]
In my case I have the same problem but I don't have such underscore problem
